pkg/pdcli/alerts/handlers.go

```diff
@@ -52,7 +56,19 @@ func GetIncidents(c client.PagerDutyClient, opts *pdApi.ListIncidentsOptions) ([
 		}
 	}
 
-	return incidents.Incidents, nil
+	for _, incident := range listOfIncidents.Incidents {
+		// When incidents are fetched for a team, do not include the incidents assigned to SilentTest
+		if areIncidentsFetchedForTeam {
```

Comment:
I would like you to add the `SilentTest` user to this conditional check, to skip incidents if assigned to that user.